migrations/1575287807413-remove_create_permissions_thuringia.js

```diff
@@ -71,16 +82,18 @@ module.exports = {
 				}
 			}
 			if (updated !== 0) {
-				info(`updating role '${role}' finished with ${updated} modifications. `
-				+ 'Removed permissions:', removedPersmissions);
+				info(
+					`updating role '${role}' finished with ${updated} modifications. ` + 'Removed permissions:',
```

Comment:
Codacy found an issue: [Unexpected string concatenation of literals.](https://app.codacy.com/manual/schul-cloud/schulcloud-server/pullRequest?prid=5989783)